### PR TITLE
Added command

### PR DESCRIPTION
**Pull request by ArtutoGamer**
Added command warn
Syntax: !warn @user reason

### CODE REVIEW

File: modules/mod.rb [hunk 1]
```diff
@@ -74,6 +74,40 @@ module Mod
 				event.respond('❌ Invalid argument. Please mention a valid user.')
 			end
 		end
+	  
+		command(:warn, description: 'Warn somebody on the server. Mod only.', usage: "#{Config.prefix}warn @user reason", min_args: 2) do |event, *kick_reason|
+			unless Helper.is_helper?(event) || Helper.is_moderator?(event) || Helper.is_developer?(event) || Helper.is_admin?(event.user)
+				event.respond("âŒ You don't have permission for that!")
```
Comment (spotlightishere) [inline modules/mod.rb]: I think your formatting here got messed up :/

File: modules/mod.rb [hunk 2]
```diff
@@ -74,6 +74,40 @@ module Mod
...
+				display = final_message.join(' ')
+				message = "You have been warned on the server **#{event.server.name}** by #{event.message.author.mention} | **#{event.message.author.display_name}**\n"
+				message << "They gave the following reason: ``#{display}``"
+				begin
+						member.pm(message)
+				rescue Discordrb::Errors::NoPermission
+						event.respond('Could not DM user about warn reason!')
+						break
+				end
+				begin
```
Comment (spotlightishere) [inline modules/mod.rb]: You can remove this entire ```
begin
 whatever
rescue
xd
end``` part as it's only for kicking.

File: modules/mod.rb [hunk 3]
```diff
@@ -74,6 +74,40 @@ module Mod
...
+					# Register for logging
+					Logging.record_action('kick', event.user, member, display)
```
Comment (spotlightishere) [inline modules/mod.rb]: Though keep this line. Might wanna change it from kick to warn too. I'm gonna have to revamp this system this weekend.

File: modules/mod.rb [hunk 4]
```diff
@@ -74,6 +74,40 @@ module Mod
...
+				rescue Discordrb::Errors::NoPermission
+					event.respond("â—â—â— The bot doesn't have permission to warn!")
+					break
+				end
+				event.respond('âœ… Warned!')
```
Comment (spotlightishere) [inline modules/mod.rb]: Perhaps here we might wanna output to the appropriate log? I can add this later myself, if not you can copy some stuff from modules/Logging.rb to here.

File: modules/mod.rb (hunk 1 above)
Comment (Seriell) [inline modules/mod.rb]: GitHub is having issues displaying them, but they show correctly outside of it, ignore

File: modules/mod.rb (hunk 2 above)
Comment (Seriell) [inline modules/mod.rb]: Keep it, some users have DMs blocked.

File: modules/mod.rb (hunk 3 above)
Comment (Seriell) [inline modules/mod.rb]: Yea change to warn

File: modules/mod.rb (hunk 2 above)
Comment (spotlightishere) [inline modules/mod.rb]: No, there's a separate one for kicking and a separate one for DMs.

Comment (Seriell) [inline modules/mod.rb]: ?

Comment (spotlightishere) [inline modules/mod.rb]: There are two begin rescue blocks, I suggested to remove the one around the logging part. Keep the one with DMs.

**Comment by spotlightishere** [comment]
Any updates on changes?

**Comment by ArtutoGamer** [comment]
Im working on it. Soon™

**Comment by ArtutoGamer** [comment]
Okay, It will require more time so, I will closing this and I'm going to open another in a future time.

**Comment by spotlightishere** [comment]
Gonna begin getting this up-to-date, with @ArtutoGamer's approval.

**Comment by spotlightishere** [comment]
welp we have an issue, I kind of merged the change in manually to like 3 branches. I'll close this now, and will begin work in the `warn` branch.

**Pull request closed without merging**